test/parallel/test-http2-invalidheaderfield.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto) { common.skip('missing crypto'); }
+
+// Check for:
+// Spaced headers
+// Psuedo headers
+// Capitalized headers
+
+const http2 = require('http2');
+const { throws } = require('assert');
+
+const server = http2.createServer(common.mustCall((req, res) => {
+
+  throws(() => {
+    res.setHeader(':path', '/');
+  }, TypeError);
+
+  throws(() => {
+    res.setHeader('t est', 123);
```

Comment:
Could you also add a test for `res.setHeader(' test', 123);` the previously missed case?

Reply:
Done.